instance_ips.go

```diff
@@ -144,3 +150,13 @@ func (c *Client) DeleteInstanceIPAddress(ctx context.Context, linodeID int, ipAd
 	err := doDELETERequest(ctx, c, e)
 	return err
 }
+
+// Function to add additional reserved IPV4 addresses to an existing linode
+func (c *Client) AddReservedIPToInstance(ctx context.Context, linodeID int, opts InstanceReserveIPOptions) (*InstanceIP, error) {
```

Comment:
nitpick: Could this endpoint method be renamed to `AssignInstanceReservedIP` for consistency?

Reply:
Changed the name to AssignInstanceReservedIP